docs/src/auth.md

```diff
@@ -148,14 +148,48 @@ var context = await browser.NewContextAsync(new()
 ```
 
 ### Code generation
+* langs: js
 
 Logging in via the UI and then reusing authentication state can be combined to
 implement **login once and run multiple scenarios**. The lifecycle looks like:
 
 1. Run tests (for example, with `npm run test`).
-1. Login via UI and retrieve authentication state.
+2. Login via UI and retrieve authentication state.
     * In Jest, this can be executed in [`globalSetup`](https://jestjs.io/docs/en/configuration#globalsetup-string).
-1. In each test, load authentication state in `beforeEach` or `beforeAll` step.
+3. In each test, load authentication state in `beforeEach` or `beforeAll` step.
+
+This approach will also **work in CI environments**, since it does not rely on any external state.
+
+### Code generation
+* langs: python
+
+Logging in via the UI and then reusing authentication state can be combined to implement **login once and run multiple scenarios**. The lifecycle looks like:
+
+1. Run tests (for example, with `pytest`).
+2. Login via UI and retrieve authentication state.
+3. In each test, load authentication state in `beforeEach` or `beforeAll` step.
+
+This approach will also **work in CI environments**, since it does not rely on any external state.
+
+### Code generation
+* langs: dotnet
+
+Logging in via the UI and then reusing authentication state can be combined to implement **login once and run multiple scenarios**. The lifecycle looks like:
+
+1. Run tests (for example, with `dotnet test`).
+2. Login via UI and retrieve authentication state.
+3. In each test, load authentication state in `beforeEach` or `beforeAll` step.
```

Comment:
In .NET with nunit it's called `SetUp` or a `TestFixture` with `SetUp`: https://docs.nunit.org/articles/nunit/writing-tests/setup-teardown/index.html